io-engine/src/bin/io-engine-client/v1/replica_cli.rs

```diff
@@ -412,6 +427,41 @@ async fn replica_unshare(
     Ok(())
 }
 
+async fn replica_resize(
+    mut ctx: Context,
+    matches: &ArgMatches,
+) -> crate::Result<()> {
+    let uuid = matches.get_one::<String>("uuid").unwrap().to_owned();
+
+    let requested_size =
+        parse_size(matches.get_one::<String>("size").ok_or_else(|| {
+            ClientError::MissingValue {
+                field: "size".to_string(),
+            }
+        })?)
+        .map_err(|s| Status::invalid_argument(format!("Bad size '{s}'")))
+        .context(GrpcStatus)?;
+
+    let _ = ctx
+        .v1
+        .replica
+        .resize_replica(v1_rpc::replica::ResizeReplicaRequest {
+            uuid: uuid.clone(),
+            requested_size: requested_size.get_bytes() as u64,
+        })
+        .await
+        .context(GrpcStatus)?;
+
+    match ctx.output {
+        OutputFormat::Json => {}
+        OutputFormat::Default => {
+            println!("replica: {} is resized", &uuid);
```

Comment:
`:` seems a bit awkward?

```suggestion
            println!("replica {} is resized", &uuid);
```